### PR TITLE
Software 0.32.1 release notes

### PR DESCRIPTION
**Pull request by jwitz**

### CODE REVIEW

**Review by danielhoherd — APPROVED** [review]

File: software/release-notes.md [hunk 1]
```diff
@@ -9,6 +9,39 @@ description: Astronomer Software release notes.
 
 0.32 is the latest stable version of Astronomer Software, while 0.30 remains the latest long-term support (LTS) release. To upgrade to 0.32, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](https://docs.astronomer.io/astro/cli/release-notes).
 
+## 0.31.1
```
Comment (rishkarajgi) [inline software/release-notes.md:12]: *0.32.1

File: software/cluster-resource-provisioning.md [hunk 2]
```diff
@@ -25,9 +25,22 @@ To change this behavior, you can change the amount of CPU and memory that an AU
             # This is a multiplying factor as a percentage of the limits. Defaults to 1
             overProvisioningFactorMem: 1
             overProvisioningFactorCPU: 1
+
+            ## (Optional) Remove components that the underprovisioning factor should not apply to
```
Comment (jwitz) [inline software/cluster-resource-provisioning.md:29]:
```suggestion
            ## (Optional) Remove components that the overprovisioning factor should not apply to
```

File: software/release-notes.md (hunk 1 above)
Comment (jwitz) [inline software/release-notes.md:12]:
```suggestion
## 0.32.1
```

**Comment by Simpcyclassy** [comment]
@jwitz update release date and we are good to go.

File: software/release-notes.md [hunk 3]
```diff
@@ -9,6 +9,39 @@ description: Astronomer Software release notes.
 
 0.32 is the latest stable version of Astronomer Software, while 0.30 remains the latest long-term support (LTS) release. To upgrade to 0.32, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](https://docs.astronomer.io/astro/cli/release-notes).
 
+## 0.32.1
+
+Release date: June 2, 2023
```
Comment (jwitz) [inline software/release-notes.md:14]:
```suggestion
Release date: June 8, 2023
```

File: software/release-notes.md [hunk 4]
```diff
@@ -9,6 +9,39 @@ description: Astronomer Software release notes.
...
+    - Flower
+  
+- You can now configure `astronomer.houston.config.deployments.overProvisioningComponents` to limit the scope of [underprovisioning](cluster-resource-provisioning.md) only to specific Airflow components.
+- Teams without any users are now automatically deleted.
+- You can now set `prometheus.config.scrape_configs.kubernetes_apiservers.tls_config.insecure_skip_verify` in the Prometheus Helm chart.
+- You can now set `astronomer.houston.config.deployments.helm.prometheus.certgenerator.extraAnnotations` in your `config.yaml` file.
+
+### Bug fixes
+
+- Fixed an issue where a service account with the Workspace Editor role could update a Deployment when it didn't have any Deployment-level permissions for the Deployment.
```
Comment (jwitz) [inline software/release-notes.md:31]:
```suggestion
- Fixed an issue where a service account with the Workspace Editor role could update a Deployment when it didn't have any Deployment-level permissions for the Deployment. 
- Fixed an issue where data for **Disk Usage** and **Platform Overview** did not appear in Grafana.
```

File: software/release-notes.md [hunk 5]
```diff
@@ -9,6 +9,45 @@ description: Astronomer Software release notes.
 
 0.32 is the latest stable version of Astronomer Software, while 0.30 remains the latest long-term support (LTS) release. To upgrade to 0.32, see [Upgrade Astronomer](upgrade-astronomer.md). For more information about Software release channels, see [Release and lifecycle policies](release-lifecycle-policy.md). To read release notes specifically for the Astro CLI, see [Astro CLI release notes](https://docs.astronomer.io/astro/cli/release-notes).
 
+## 0.32.1
+
+Release date: June 8, 2023
```
Comment (rishkarajgi) [inline software/release-notes.md:14]: June 12, 2023

File: software/release-notes.md [hunk 6]
```diff
@@ -9,6 +9,45 @@ description: Astronomer Software release notes.
...
+### Bug fixes
+
+- Fixed an issue where `git-sync-relay` containers wouldn't restart as expected after being terminated.
+- Fixed an issue where a service account with the Workspace Editor role could update a Deployment when it didn't have any Deployment-level permissions for the Deployment. 
+- Fixed an issue where data for **Disk Usage** and **Platform Overview** did not appear in Grafana.
+- System Admins can no longer change a user's system role if the user is imported to Astronomer through an IdP group.
+- Fixed an issue where you could not create a new Deployment from the Cloud UI if you updated its scheduler count using the text-based input field. 
+- Fixed an issue where container status and usage did not appear in the **Metrics** tab for Deployments with pre-created namespaces.
+- Fixed an issue where resource requests configured from the Software UI could get out of sync with the Houston database.
+- Fixed an issue where underprovisioning factors did not persist in the Houston database.
```
Comment (rishkarajgi) [inline software/release-notes.md:39]: Fixed an issue where updated component resources configuration did not persist in the Houston database on applying the overprovisioning factor

File: software/release-notes.md [hunk 7]
```diff
@@ -34,7 +34,7 @@ Release date: June 8, 2023
 - Fixed an issue where you could not create a new Deployment from the Cloud UI if you updated its scheduler count using the text-based input field. 
 - Fixed an issue where container status and usage did not appear in the **Metrics** tab for Deployments with pre-created namespaces.
 - Fixed an issue where resource requests configured from the Software UI could get out of sync with the Houston database.
-- Fixed an issue where underprovisioning factors did not persist in the Houston database.
+- Fixed an issue where where updating a Deployment's resource configuration did not persist in the Houston database when that Deployment had overprovisiniong enabled.
```
Comment (rishkarajgi) [inline software/release-notes.md:39]: *overprovisioning

File: software/release-notes.md [hunk 8]
```diff
@@ -9,6 +9,49 @@ description: Astronomer Software release notes.
...
+### Bug fixes
+
+- Fixed an issue where `git-sync-relay` containers wouldn't restart as expected after being terminated.
+- Fixed an issue where a service account with the Workspace Editor role could update a Deployment when it didn't have any Deployment-level permissions for the Deployment. 
+- Fixed an issue where data for **Disk Usage** and **Platform Overview** did not appear in Grafana.
+- System Admins can no longer change a user's system role if the user is imported to Astronomer through an IdP group.
+- Fixed an issue where you could not create a new Deployment from the Cloud UI if you updated its scheduler count using the text-based input field. 
+- Fixed an issue where container status and usage did not appear in the **Metrics** tab for Deployments with pre-created namespaces.
+- Fixed an issue where resource requests configured from the Software UI could get out of sync with the Houston database.
+- Fixed an issue where where updating a Deployment's resource configuration did not persist in the Houston database when that Deployment had overprovisiniong enabled.
```
Comment (jwitz) [inline software/release-notes.md:43]:
```suggestion
- Fixed an issue where where updating a Deployment's resource configuration did not persist in the Houston database when that Deployment had overprovisioning enabled.
```

**Comment by danielhoherd** [comment]
I think we should just not use the term "underprovisioning" anywhere. That term was never used during the design of the feature. It's a bit of a hard concept to understand, but "overprovisioning" is the right term. This is from the perspective of the deployment, not the node. We are provisioning more deployments than the node would otherwise handle, meaning we are packing more deployments onto a single node. This does mean the node is "underprovisioned" but that is not the lever we are working on. We are working with deployments, not nodes, so even though the node ends up being "underprovisioned" the thing we are actively doing is "overprovisioning" the deployments.

File: software/release-notes.md [hunk 9]
```diff
@@ -9,10 +9,54 @@ description: Astronomer Software release notes.
...
+- You can now set `prometheus.config.scrape_configs.kubernetes_apiservers.tls_config.insecure_skip_verify` in the Prometheus Helm chart.
+- You can now set `astronomer.houston.config.deployments.helm.prometheus.certgenerator.extraAnnotations` in your `config.yaml` file.
+- You can now configure credentials for a registry backend as Kubernetes secrets in your `config.yaml` file. See [Configure a registry backend](registry-backend.md).
+
+### Bug fixes
+
+- Fixed an issue where `git-sync-relay` containers wouldn't restart as expected after being terminated.
+- Fixed an issue where a service account with the Workspace Editor role could update a Deployment when it didn't have any Deployment-level permissions for the Deployment. 
+- Fixed an issue where data for **Disk Usage** and **Platform Overview** did not appear in Grafana.
+- System Admins can no longer change a user's system role if the user is imported to Astronomer through an IdP group.
```
Comment (Simpcyclassy) [inline software/release-notes.md:36]: When manageSystemPermissionsViaIdpGroups is true

**Review by rishkarajgi — APPROVED** [review]
some pending comments. Rest lgtm

File: software/release-notes.md [hunk 10]
```diff
@@ -9,10 +9,54 @@ description: Astronomer Software release notes.
...
+
+### Additional improvements
+
+- [Underprovisioning](cluster-resource-provisioning.md) now also applies to the following components:
+
+    - PGBouncer
+    - Statsd
+    - Flower
+  
+- You can now configure `astronomer.houston.config.deployments.overProvisioningComponents` to limit the scope of [underprovisioning](cluster-resource-provisioning.md) only to specific Airflow components.
```
Comment (jwitz) [inline software/release-notes.md:24]:
```suggestion
- You can now configure `astronomer.houston.config.deployments.overProvisioningComponents` to limit the scope of [overprovisioning](cluster-resource-provisioning.md) only to specific Airflow components.
```

**Review by Simpcyclassy — APPROVED** [review]

**Comment by jwitz** [comment]
I think we can remove the concept of "underprovisioning" and just clarify what we mean by "overprovisioning". The lack of specificity is better than explaining things in the exact opposite terms of the Helm configuration name.

**Pull request merged**